Razor/Map/MapWindow.cs

```diff
@@ -52,8 +52,8 @@ public MapWindow()
             // Required for Windows Form Designer support
             //
             InitializeComponent();
-            this.ContextMenu = new ContextMenu();
-            this.ContextMenu.Popup += new EventHandler(ContextMenu_Popup);
+            this.ContextMenuStrip = new ContextMenuStrip();
+            this.ContextMenuStrip.Opening += ContextMenu_Popup;

```

Comment:
This breaks the right click opening on UOPS. There isn't a popup event like before, so I need to find a solution.